python/common/org/python/types/Float.java

```diff
@@ -578,7 +588,22 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
         __doc__ = ""
     )
     public org.python.Object __round__(org.python.Object ndigits) {
-        throw new org.python.exceptions.NotImplementedError("float.__round__() has not been implemented.");
+        /*if((org.python.types.Int)ndigits != new org.python.types.Int(0)) {
```

Comment:
Hey!
Yes, this was not implemented properly which is why I commented it out. I'll be sending in more commits soon, which would rectify this.